### ISSUE TITLE
use for adding metadata to data products

### ISSUE THREAD

**Issue by heikekonow** [eurec4a/meta#5]
My use case for the eurec4a meta file would be to read it and use the information to automatically add the relevant information as attributes to netcdf files of data products

**Comment by d70-t** [comment]
Do you have specific ideas, which metadata you would like to have available? Things which come to my mind at the moment are
* Associated people, institutions, platforms, funders etc...
* Standardized variable names
* Units
* Links to documentation

**Comment by heikekonow** [comment]
Yes, those were the things I was thinking about. Just to make it easier for people generating data sets to use the correct contact info, necessary credits, etc.

Let's see if I'll think of something else to add here.
